pkg/controller/controller.go

```diff
@@ -157,10 +157,12 @@ const (
 func (c *controller) reconcileBroker(broker *v1alpha1.Broker) {
 	glog.V(4).Infof("Processing Broker %v", broker.Name)
 
-	username, password, err := GetAuthCredentialsFromBroker(c.kubeClient, broker)
+	username, password, err := getAuthCredentialsFromBroker(c.kubeClient, broker)
 	if err != nil {
-		glog.Errorf("Error getting broker auth credentials for broker %v: %v", broker.Name, err)
-		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason, errorFetchingCatalogMessage)
+		err:= fmt.Errorf("Error getting broker auth credentials for broker %v: %v", broker.Name, err)
+		glog.Info(err)
+		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason,
```

Comment:
In this case while I'm ok with including the broker name in the condition, technically its not needed since this condition is within the scope of the broker object - so people should know which one it is w/o the name. I think including the error string is the critical piece.  Having the name in the log msg though is good.